src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import { Provider } from "react-redux";
+import MainPage from "./components/MainPage";
+import Footer from "./components/Footer";
+import store from "./redux/reduxStore";
+
+const App = () => {
+	return (
+		<MainPage>
+			<Footer />
+		</MainPage>
+	);
+};
+
+const WrappedApp = () => {
+	return (
+		<Provider store={store}>
+			<App />
+		</Provider>
+	);
+};
```

Comment:
It's a bad experience to save several components in 1 file. 1 file - 1 component